### PR TITLE
KVM v11.3.3 release

### PR DESCRIPTION
**Pull request by tomahawk28**
Toward https://github.com/giantswarm/giantswarm/issues/11510

Adding 11.3.3 into KVM release.

### CODE REVIEW

**Comment by tomahawk28** [comment]
I'm not sure I could deactivate 11.3.2 KVM release because it was released just today. 
Let's stay that active that for now.

**Comment by tomahawk28** [comment]
@cokiengchiara Would you please look into changelog?

File: kvm/v11.3.3/release-notes.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## :zap: Giant Swarm Release 11.3.3 for KVM :zap:
+
+**If you are upgrading from 11.3.1 or 11,3,2 upgrading to this release will not roll your nodes. It will only update the apps.**
+
+This release includes a security fix from helm 2.16.8.
```
Comment (cokiengchiara) [inline kvm/v11.3.3/release-notes.md:5]:
```suggestion
This release upgrades the chart-operator component to Helm 2.16.8, addressing a high severity security vulnerability identified in Go's `crypto` package.
```

File: kvm/v11.3.3/release-notes.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+## :zap: Giant Swarm Release 11.3.3 for KVM :zap:
+
+**If you are upgrading from 11.3.1 or 11,3,2 upgrading to this release will not roll your nodes. It will only update the apps.**
+
+This release upgrades the chart-operator component to Helm 2.16.8, addressing a high severity security vulnerability identified in Go's `crypto` package.
```
Comment (sslavic) [inline kvm/v11.3.3/release-notes.md:5]:
```suggestion
This release upgrades the chart-operator component to Helm 2.16.8, addressing a security vulnerability identified in Go's `crypto` package.
```

Removing qualification, unless @stone-z classifies it better.

**Comment by cokiengchiara** [comment]
@giantswarm/sig-security could you review the release notes? @sslavic suggests to tone down to reflect actual severity of issue ✌️ 

not urgent, we could do it next week

**Comment by tomahawk28** [comment]
Sorry for hassles, but looks like this CVE is only for 32-bit architecture. I missed that part. 
We can safely close this. 
Sorry! @sslavic, @cokiengchiara

**Pull request closed without merging**